OnlinerTask/OnlinerTask.BLL/Services/ElasticSearch/ProductLogger/Implementations/OnlinerProductLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using Nest;
+using OnlinerTask.BLL.Services.ElasticSearch.ProductLogger.ClientsFabric;
+using OnlinerTask.BLL.Services.ElasticSearch.ProductLogger.ConnectionFabric;
+using OnlinerTask.Data.Resources;
+using OnlinerTask.Data.SearchModels;
+
+namespace OnlinerTask.BLL.Services.ElasticSearch.ProductLogger.Implementations
+{
+    public class OnlinerProductLogger : IProductLogger
+    {
+        private readonly ElasticClient elasticClient;
+
+        public OnlinerProductLogger(IClientsFabric clientsFabric, IConnectionFabric connectionFabric)
+        {
+            var settings = connectionFabric.CreateConnection(Configurations.ElasticSearchUrl, Configurations.ProductLogIndex);
+            elasticClient = clientsFabric.CreateClient(settings, Configurations.ProductLogIndex);
+        }
+
+        public async Task LogAdding(int productId, ProductModel productModel)
```

Comment:
Names are not so good. Also think about common logging mechanism. On each model type you should not implement another one method.